pkg/api-server/testdata/resources/inspect/dataplanes/_rules/meshgateway.golden.json

```diff
@@ -12,6 +12,40 @@
  "rules": [
   {
    "fromRules": [],
+   "toResourceRules": [
+    {
+     "conf": [
+      {
+       "connectionTimeout": "2s",
+       "idleTimeout": "20s",
+       "http": {
+        "requestTimeout": "10s"
+       }
+      }
+     ],
+     "origin": [
+      {
+       "resourceMeta": {
+        "labels": {
+         "k8s.kuma.io/namespace": "kuma-demo",
+         "kuma.io/display-name": "mt-on-gateway"
+        },
+        "mesh": "default",
+        "name": "mt-on-gateway",
+        "type": "Mesh"
+       },
+       "ruleIndex": 0
+      }
+     ],
+     "resourceMeta": {
+      "labels": {},
+      "mesh": "",
+      "name": "default",
+      "type": "Mesh"
+     },
+     "resourceSectionName": ""
```

Comment:
and also we need a test when `resourceSectionName` is set

Reply:
added